styles/global.css

```diff
@@ -3,30 +3,30 @@
 
 /* Global styles */
 :root {
-  --primary: #2f6ed3;
-  --primary-light: #f1f7ff;
-  --secondary: #27374c;
-  --secondary-light: #5842c4;
+  --primary: #2f6ed3 !important;
+  --primary-light: #f1f7ff !important;
+  --secondary: #27374c !important;
+  --secondary-light: #5842c4 !important;
```

Comment:
using important is bad practice try to find a work around this

Reply:
oh right i had to remove these, forgot about it, thanks for pointing it out